source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,126 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
+works to make this as simple as possible, using 
+`JSON Schema <https://docs.mongodb.com/manual/reference/operator/query/jsonSchema/#json-schema>`_ 
+for validation during insert and update operations. Within the context of Field Level Encryption,
+JSON Schema is used to specify several configuration details, allowing the drivers
+to automatically encrypt and decrypt fields. 
+
+* With CSFLE enabled, the driver will automatically encrypt and decrypt fields if the following properties are specified in the JSON Schema:
+    - The encryption algorithm (deterministic or random)
+    - The data key
+    - The BSON Type
+* You can use a single data key for all encrypted fields or you can use individual data keys that are specific to each field.
+
+
+The following table represents the fields of data that MedcoMD will be using in their Medical Care Management System. Each field
+will be either unencrypted or encrypted. Encrypted fields will be encrypted through `Deterministic Encryption <https://docs.mongodb.com/manual/core/security-client-side-encryption/#deterministic-encryption>`_
+or `Random encryption <https://docs.mongodb.com/manual/core/security-client-side-encryption/#random-encryption>`_.
+
+.. table:: 
+    :widths: 50 50
+
+    +-----------------------------+---------------------+----------------------------------------------+
+    |Field type                   |Encryption Algorithm | BSON Type                                    |
+    +=============================+=====================+==============================================+
+    | Name                        | Non-Encrypted       | String                                       |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | SSN                         | Deterministic       |  Int                                         |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Blood Type                  | Random              |  String                                      |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Medical Records             | Random              |  Array                                       |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Insurance: Policy Number    | Deterministic       |  Int (embedded inside `insurance` object)    |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Insurance: Provider         | Non-Encrypted       |  String (embedded inside `insurance` object) |
+    +-----------------------------+---------------------+----------------------------------------------+
+
+You will use the secure client, created in an earlier section, by passing it a JSON object that contains your schema. 
```

Comment:
Nitpick: [style guide: use present tense](https://docs.mongodb.com/meta/style-guide/writing/use-present-tense/)

E.g.
```
You will use the secure client... -> Pass the schema as a JSON object to the secure client to specify the encrypted field properties.
```

Reply:
Great idea, I've fixed it now